upi/vsphere/machine/ignition.tf

```diff
@@ -24,12 +24,12 @@ data "ignition_file" "static_ip" {
   count = "${var.instance_count}"
 
   path       = "/etc/NetworkManager/system-connections/eth0"
-  mode       = "0600"
+  mode       = "384"
 
   content {
     content = <<EOF
 [connection]
-id=Wired connnection 1
+id=Wired connection 1
 uuid=27afa607-ee36-43f0-b8c3-9d245cdc4bb3
```

Comment:
I wonder whether there is a common way to autogenerate this with terraform? Hardcoding the uuid seems weird.

Reply:
Right, I think https://www.terraform.io/docs/configuration/functions/uuid.html can be used here